config/build/rollup.config.js

```diff
@@ -152,7 +154,38 @@ const createConfig = (
                 extract: "index.css",
                 minimize: true,
                 sourceMap: true,
-                plugins: [postcssImport()],
+                plugins: [
+                    postcssImport(),
+                    // The postcssUrl() plugin is used to re-write relative
+                    // paths in `url(...)` statements in our CSS to be relative
+                    // to the `dist/` output folder. It also copies the files
+                    // referenced by these `url()`s to the `dist/` folder also.
+                    // This is mostly for the Symbola fonts referenced by
+                    // Mathquill CSS.
+                    //
+                    // CAUTION: This PostCSS plugin is patched (using `pnpm
+                    // patch`). You can see the patch in this repo at
+                    // `patches/postcss-url.patch`. If you ever upgrade to a
+                    // newer version of this plugin, please double-check that:
+                    //   a) the font file copying still works (ie. the fonts are
+                    //      copied to the `math-input/dist/assets` folder and/or
+                    //   b) if the patch is even needed anymore)
```

Comment:
Was this meant to be this?
```suggestion
                    //   a) the font file copying still works (ie. the fonts are
                    //      copied to the `math-input/dist/assets` folder) and/or
                    //   b) if the patch is even needed anymore
```